backend/src/index.js

```diff
@@ -9,10 +9,16 @@
 const PORT = process.env.PORT || 3000;
 
 const app = require( 'express' )();
+const { promisify } = require( 'util' );
 const http = require( 'http' ).Server( app );
 const io = require( 'socket.io' )( http );
+const Redis = require( 'ioredis' );
+const redisAdapter = require( 'socket.io-redis' );
+io.adapter( redisAdapter( { host: 'localhost', port: 6379 } ) );
 
-const db = new Map();
+const client = new Redis();
+const getAsync = promisify( client.get ).bind( client );
+const setAsync = promisify( client.set ).bind( client );
```

Comment:
Can `getAsync` and `setAsync` be a little bit better named?

Reply:
And the `async` part should be omitted. The original functions where async too.

Reply:
I'd propose something like `fetchUsers` and `storeUsers`.

Reply:
Good proposition. I've already updated it.